.travis.yml

```diff
@@ -21,10 +26,11 @@ install:
   - conda update -q conda
   - conda info -a
 
-  - conda env create -q -n skorch-env -f environment.yml python=${TRAVIS_PYTHON_VERSION}
+  - conda create -y -n skorch-env python=${TRAVIS_PYTHON_VERSION} scikit-learn tqdm
```

Comment:
Why drop the `environment.yml`? Isn't CI with a fixed environment preferable?

Reply:
The current version of `environment.yml` file fixes the python version. This meant all test runs used python 3.6. 

This PR was updated to use the `requirements.txt` to install dependencies.